README.md

```diff
@@ -5,3 +5,22 @@
 [![Build Status](https://travis-ci.org/taskcluster/taskcluster-vcs.svg?branch=master)](https://travis-ci.org/taskcluster/taskcluster-vcs)
 
 [Read the docs](http://tc-vcs.readthedocs.org/en/latest/)
+
+# Deploying new Docker images
+
+
+```
+# Edit `Dockerfile` and `src/task.js` to update the version number.
+docker bulid -t taskcluster/taskcluster-vcs:$REVISION .
```

Comment:
Hrm, are we tagging the image with revision, or version number of the taskcluster-vcs package?